File: Natom.Petshop.Gestion.Backend/Natom.Petshop.Gestion.Entities/DTO/Stock/MovimientoStockDTO.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Natom.Petshop.Gestion.Entities.DTO.Stock
{
    public class MovimientoStockDTO
    {
        [JsonProperty("deposito_encrypted_id")]
        public string DepositoEncryptedId { get; set; }

        [JsonProperty("producto_encrypted_id")]
        public string ProductoEncryptedId { get; set; }

        [JsonProperty("tipo")]
        public string Tipo { get; set; }

        [JsonProperty("cantidad")]
        public decimal Cantidad { get; set; }

        [JsonProperty("usuario_encrypted_id")]
        public string UsuarioEncryptedId { get; set; }

        [JsonProperty("observaciones")]
        public string Observaciones { get; set; }

        [JsonProperty("esCompra")]
        public bool EsCompra { get; set; }

        [JsonProperty("proveedor_encrypted_id")]
        public string ProveedorEncryptedId { get; set; }

        [JsonProperty("costoUnitario")]
        public decimal? CostoUnitario { get; set; }
    }
}

File: Natom.Petshop.Gestion.Frontend/src/app/classes/dto/stock/movimiento-stock.dto.ts
export class MovimientoStockDTO {
    public deposito_encrypted_id: string;
    public producto_encrypted_id: string;
    public tipo: string;
    public cantidad: number;
    public usuario_encrypted_id: string;
    public usuarioNombre: string;
    public observaciones: string;
    public esCompra: boolean;
    public proveedor_encrypted_id: string;
    public costoUnitario: number;
}